# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipvsadm/old_suffix.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-02-03 19:21:42.000000000 -0300
+++ ./Makefile	2005-02-03 19:23:32.000000000 -0300
@@ -91,15 +91,15 @@
 
 install:        ipvsadm
 		if [ ! -d $(SBIN) ]; then $(MKDIR) -p $(SBIN); fi
-		$(INSTALL) -m 0755 -s ipvsadm $(SBIN)
-		$(INSTALL) -m 0755 ipvsadm-save $(SBIN)
-		$(INSTALL) -m 0755 ipvsadm-restore $(SBIN)
+		$(INSTALL) -m 0755 -s ipvsadm $(SBIN)/ipvsadm.old
+		$(INSTALL) -m 0755 ipvsadm-save $(SBIN)/ipvsadm-save.old
+		$(INSTALL) -m 0755 ipvsadm-restore $(SBIN)/ipvsadm-restore.old
 		[ -d $(MAN) ] || $(MKDIR) -p $(MAN)
-		$(INSTALL) -m 0644 ipvsadm.8 $(MAN)
-		$(INSTALL) -m 0644 ipvsadm-save.8 $(MAN)
-		$(INSTALL) -m 0644 ipvsadm-restore.8 $(MAN)
+		$(INSTALL) -m 0644 ipvsadm.8 $(MAN)/ipvsadm.old.8
+		$(INSTALL) -m 0644 ipvsadm-save.8 $(MAN)/ipvsadm-save.old.8
+		$(INSTALL) -m 0644 ipvsadm-restore.8 $(MAN)/ipvsadm-restore.old.8
 		if [ -d $(INIT) ]; then \
-		  $(INSTALL) -m 0755 ipvsadm.sh $(INIT)/ipvsadm; \
+		  $(INSTALL) -m 0755 ipvsadm.sh $(INIT)/ipvsadm.old; \
 		fi
 
 clean:
